controllers/product.controller.js

```diff
@@ -14,7 +14,7 @@ exports.create = async (req, res) => {
     try {
         const {name, description, price, category, color} = req.body;
         const product = await Product.create({name, description, price, category, color});
-        res.send(product);
+        res.status(201).send(product);
```

Comment:
You will need to change the test as well 
https://github.com/yhod/jello-node/blob/main/api/__tests/product.route.test.js#L25

Reply:
Done